admin-dev/themes/new-theme/js/pages/order/view/order-view-page.js

```diff
@@ -138,7 +140,15 @@ export default class OrderViewPage {
   listenForProductDelete() {
     $(OrderViewPageMap.productDeleteBtn)
       .off('click')
-      .on('click', event => this.orderProductManager.handleDeleteProductEvent(event));
+      .on('click', event => {
+        this.orderProductManager.handleDeleteProductEvent(event);
+      }
+    );
+  }
+
+  resetToolTips() {
```

Comment:
Thank you for this PR :)
I want to ask you 2 questions:
- Isn't it better to put this function in product renderer and call it `this.productRenderer.resetTooltips()` here?
- What about compiled assets? should I commit them also when I create a front end PR?
Thank you very much for you time and answer.

Reply:
Hello @arouiadib 

- I still prefered to put this method in the same file than others listeners resets, I was like "Why putting it in another file if every listeners are in this one ?
- Compiled assets needs to be commited on every branch before 1.7.8.x, after 1.7.7, we decided to stop commiting assets as it is an error, cause too much useless rebases and except when you'r building a library, the best practice is to have source files only on a repository.

For the first point, maybe the PrestaShop's team will decline it, let's see what they think about it too :)

You'r welcome! 😄